packages/react-devtools-extensions/build.js

```diff
@@ -56,7 +56,14 @@ const build = async (tempPath, manifestPath) => {
   const binPath = join(tempPath, 'bin');
   const zipPath = join(tempPath, 'zip');
 
-  const webpackPath = join(__dirname, 'node_modules', '.bin', 'webpack');
+  const webpackPath = join(
+    __dirname,
+    '..',
```

Comment:
changed it back as it was before https://github.com/facebook/react/pull/26887/files#diff-b6d5a082026c8873e9e518725027610b6936836db8bd4af2a7fa15fbdb563b7eL59